jina/peapods/runtimes/gateway/prefetch.py

```diff
@@ -16,34 +16,42 @@
 class PrefetchCaller:
     """An async zmq request sender to be used in the Gateway"""
 
-    def __init__(self, args: argparse.Namespace, zmqlet: 'AsyncZmqlet'):
+    def __init__(
+        self,
+        args: argparse.Namespace,
+        zmqlet: 'AsyncZmqlet' = None,
+        grpclet: 'Grpclet' = None,
```

Comment:
We can probably accept 1 mandatory arg `iolet` here, which is `Union[AsyncZmqlet, Grpclet]`?